src/containerapp/azext_containerapp/commands.py

```diff
@@ -24,6 +26,7 @@ def load_command_table(self, args):
         g.custom_command('update', 'update_containerapp', supports_no_wait=True, exception_handler=ex_handler_factory(), table_transformer=transform_containerapp_output, transform=transform_sensitive_values)
         g.custom_command('delete', 'delete_containerapp', supports_no_wait=True, confirmation=True, exception_handler=ex_handler_factory())
         g.custom_command('up', 'containerapp_up', supports_no_wait=False, exception_handler=ex_handler_factory())
+        g.custom_command('debug', 'containerapp_debug', is_preview=True, validator=validate_ssh)
```

Comment:
as offline discussion, do we still stick to these validations that might be unnecessary, or are we going to turn to our own validations?

Reply:
Since the validation typically takes a while, you might want to show something like "Validating..." before it starts to let users know what is going on. Also probably a "Connecting..." before sending the api calls to debug endpoint